offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/OfferToPDFConverter.groovy

```diff
@@ -340,19 +346,6 @@ class OfferToPDFConverter implements OfferExporter {
         return netSum
     }
 
-    double calculateOverheadSum(List<ProductItem> productItems) {
-        double overheadSum
-        productItems.each {
-            if (it.product.class in productGroupClasses[ProductGroups.DATA_MANAGEMENT]) {
-                overheadSum = 0
-        }
-            else {
-                overheadSum += it.quantity * it.product.unitPrice * offer.overheadRatio
-            }
-        }
-        return overheadSum
-    }
-
```

Comment:
This method is not necessary anymore, since we don't list the subtotal overhead costs for each productcategory anymore. The total overhead costs was already stored in the offer and can be retrieved there.